v2/anomalydetection/backend-setup.mdx

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 3
+title: Backend Setup
+hide_title: true
+---
+
+# Setting up the Backend
+
+In order to use the anomaly detection service, a request to the following enpoint needs to be made:
+
+```typescript
+POST https://security-us-east-1.aws.supertokens.io/v1/security
+```
+
+You can view the HTTP API reference for this endpoint [here](#somelinkhere).
+
+### Retrieving the Request ID
+
+The reuqest ID that has been passed from the frontend is available in the `requestId` property of the input body from each API override.
+
+Below is an example of how to retrieve the request ID from the input body:
+
+```typescript
+const generatePasswordResetTokenPOST = async function (input) {
+    // ... your code here ...
+    const requestId = (await input.options.req.getJSONBody()).requestId;
+    // ... your code here ...
+}
+```
+
+### Making the Request to the Anomaly Detection endpoint
+
+The request body is a JSON object that contains the following properties (all fields are optional):
+
+- `securityServiceRequestId`: The request ID that has been generated on the frontend.
+- `bruteForce`: An array of brute force checks that have been configured on the frontend.
+```typescript
+type BruteForceCheck = {
+    keyStr: string;
+    maxRequests: {
+                limit: number;
+                perTimeIntervalMS: number;
+        }[];
+    }
+}
```

Comment:
need to explain this with examples. Also need to show how it's used in case of passwordless and other reicpes - that a prefix needs to be added to the key to scope the request to that API.

Also need to explain that the key is can be the email or IP based or a combination of the two based on what kind of brute force check you want to do

Reply:
Im thinking a better structure for the doc would be to have a separate page for each type of feature, and explain in that what needs to be done to enable that check. What do you think?

Reply:
i think that's a good idea, but maybe when there is more to show ?
Atm the only think that is really configurable is this. The rest are optional fields that return a value or null - that is the extent of the configurability.

Reply:
done